benchmarks/benchmark_serving.py

```diff
@@ -98,6 +98,7 @@ class InputRequest:
   prompt_len: int = 0
   output: str = ""
   output_len: int = 0
+  sample_idx: int = -1
```

Comment:
There is right mapping between request and output in benchmark. what is the purpose of sample_idx? Will it help you find the related request easily?

Reply:
+1, your goal is to be able to print the input prompt for the output for debugging right?

Reply:
My goal is to be able to locate the original order / index from the original dataset file. 

Say, if there are 10k data samples in the original dataset file, after randomly sampling, being passed as input request into server, the requests are returned based on decode completions. Currently we save the prompt, original result, and generated result in the request output file. If there are other metadata I am interested in checking in the original dataset file. How can I locate them?

Reply:
agree, it's hard to locate original dateset. Please also feel free to add the important metadata into the result.